src/client/containers/PullRequestSummary/BuildSection/index.js

```diff
@@ -1,34 +1,26 @@
 /* @flow */
 
-import React, { PureComponent } from 'react'
+import React from 'react'
+import PureComponent from 'components/PureComponent'
 import { connect } from 'react-redux'
 import { fetchLatestBuildsForRevision } from 'ducks/bfstats/actions'
 import { getPullRequest } from 'ducks/pullrequests/selectors'
 import { createSelector } from 'reselect'
 import Builds from './Builds'
 
-type BuildType = {
-  builder: {
-    name: string
-  },
-  result: number
-}
-
 type PropsType = {
-  id: string,
   repository: string,
   dispatch: Function,
   revision: string,
-  builds: Array<BuildType>,
 }
 
 export const getPRSourceStamp = (state: Object, props: Object): Object =>
   createSelector(
     getPullRequest,
-    (pr) => ({
+    pr => ({
```

Comment:
Just a small comment: I'm not sure our style rules are consistent with these parentheses. Sometimes single arguments have them, sometimes they don't.